### ISSUE TITLE
Error: no matches for kind "ServiceMonitor"

### ISSUE THREAD

**Issue by marquesds** [riskfocus/helm-charts-public#4]
Hi! I'm trying to install Flink on my Kubernetes cluster with this command:
`helm install my-flink --namespace my-namespace riskfocus/flink --version 0.1.10`

And I'm receiving the error below:
```
Error: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "ServiceMonitor" in version "monitoring.coreos.com/v1"
```

**Comment by vkropotko** [comment]
Hi @marquesds!
It was an issue with default values, by default the property `prometheus.serviceMonitor.enabled` was enabled that is why it tried to create Prometheus ServiceMonitor. Fixed in 0.1.11

**Comment by marquesds** [comment]
Thanks @vkropotko! This worked like a charm.
